src/assets/style.css

```diff
@@ -44,6 +44,7 @@
 }
 
 .playkit-subtitles {
+  z-index: 10;
```

Comment:
I feel like the changes should be applied only for audio-entries. Also, need check how it behaves with dual-screen plugin that manage caption z-index as well  
https://github.com/kaltura/playkit-js-dual-screen/blob/e98dc19c8d19d8da36d5c0c9b2f07d29ad85612b/src/styles/global.scss#L1

Reply:
could you elaborate on the dual-screen ?
because there's title/descreption on the media like audio entries,
i assume you're talking about the child video. what is the expected behavior with regards to the caption? it's supposed to be behind the captions still no?

Reply:
either ways (z-index =1 or more) the captions would be in front of the 2 videos.